core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -550,9 +550,16 @@ oppia.factory('explorationRightsService', [
         var that = this;
 
         requestParams.version = explorationData.data.version;
-        var explorationRightsUrl = (
-          '/createhandler/rights/' + explorationData.explorationId);
-        $http.put(explorationRightsUrl, requestParams).then(function(response) {
+        var requestUrl = '';
```

Comment:
To clarify that last part: I meant explicitly passing in the individual args to the new frontend functions, i.e. doing something like function(newStatus, userId, ...) (this is just an example) rather than function(bigBlobOfStuff).